test/celeritas/field/FieldPropagator.test.cc

```diff
@@ -837,8 +837,8 @@ TEST_F(TwoBoxTest, electron_step_endpoint)
         Real3     start_pos{-5 + dx, 0, 0};
         axpy(real_type(-1), first_pos, &start_pos);
 
-        auto geo       = this->init_geo(start_pos, {0, 1, 0});
-        auto stepper   = make_mag_field_stepper<DiagnosticDPStepper>(
+        auto geo     = this->init_geo(start_pos, {0, 1, 0});
+        auto stepper = make_mag_field_stepper<DiagnosticDPStepper>(
```

Comment:
I think we must be using different clang-format versions... :( I want to get rid of the equal sign indentation for this reason.

Reply:
I am not sure I understand, do you prefer how it is in green?

Reply:
I personally care less how it's formatted and more that the line doesn't change if the code doesn't change. 😞  which is the case here... don't worry about reverting it though. The point of an autoformatter is to save us from having to deal with spaces 😬